.github/workflows/no-response.yml

```diff
@@ -0,0 +1,23 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 0 * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb
+        with:
+          token: ${{ github.token }}
+          responseRequiredLabel: needs-more-info
```

Comment:
@IEvangelist I think it works on this label.

Reply:
Yeah, it basically works just like the existing bot does. The bot app that was doing this for us points to this action as its replacement.